src/arrow.rs

```diff
@@ -0,0 +1,111 @@
+use bevy::{math::DVec2, prelude::*};
+use bevy_vello::{prelude::*, vello::kurbo::Shape};
+
+use super::VelloVector;
+
+#[derive(Component, Default, Debug, Clone, Copy)]
+pub struct VelloArrow {
+    pub head: ArrowHead,
+
+    // parent
+    pub position: kurbo::Vec2,
+    pub angle: f64, // radians
+
+    // settings
+    pub offset: kurbo::Vec2,
```

Comment:
using `kurbo::Vec2` instead of `DVec2` because of convienience with converting to `kurbo::Point` for `shape()`. This leads to some silly conversion in `VelloArrow::attach()` but why are we using DVec2 to begin with? seems like kurbo types are better suited for this.

Reply:
Mostly for the friendly use case for bevy user as this package is targeted more towards bevy users with minimal vello knowledge.